### ISSUE TITLE
Floating geomtry, a plant, and a flag in Winterhold Imperial Camp

### ISSUE THREAD

**Issue by themaster567** [ForgottenGlory/Living-Skyrim-2#483]
**LS Version**
Latest version as of two days ago.

**Describe the bug**
Several cases of various floating things.

**To Reproduce**
Visit the Winterhold Imperial Camp.

**Expected behavior**
The objects should not be floating.

**Screenshots**
![SkyrimSE_Hi3dQeiheC](https://user-images.githubusercontent.com/8242023/96950387-2dced600-14b8-11eb-8d7d-c5f1a3840e37.jpg)
![SkyrimSE_KWeCfsyHVi](https://user-images.githubusercontent.com/8242023/96950392-30c9c680-14b8-11eb-854b-23188be6680b.jpg)
![SkyrimSE_Bviu1Lkh6P](https://user-images.githubusercontent.com/8242023/96950395-32938a00-14b8-11eb-9c3d-b154e93099be.jpg)

**Additional context**

**Comment by TwistedModding** [comment]
When you get the chance can you reupload pictures of these with the items selected in the console?

**Comment by themaster567** [comment]
Absolutely. I'll do that tomorrow morning. I'm also about to file a seperate report regarding competing water types in Morthal. I'll make sure to highlight those.

**Comment by TwistedModding** [comment]
With water stuff you cant select it in the console. Open up the dyndolod mcm menu and take a screenshot of the "where am i" (or something named liked that) cords and post it to the github.

**Comment by themaster567** [comment]
I wasn't able to select the rocks and one of the snowdrifts, but here are pictures of what I could highlight:
![SkyrimSE_RLt0alsojz](https://user-images.githubusercontent.com/8242023/97020674-d6645080-151f-11eb-972e-79c482843aea.jpg)
![SkyrimSE_KcARPpyfyK](https://user-images.githubusercontent.com/8242023/97020684-d95f4100-151f-11eb-8015-6d589fa1168d.jpg)
![SkyrimSE_JdfYkEQ8aQ](https://user-images.githubusercontent.com/8242023/97020697-de23f500-151f-11eb-9e9f-abf9b037ae38.jpg)

Here's the snowdrift that I couldn't highlight for some reason:
![SkyrimSE_IJLx2qzTBY](https://user-images.githubusercontent.com/8242023/97020730-ea0fb700-151f-11eb-9576-37160881fccf.jpg)

**Comment by TwistedModding** [comment]
Anything you can't select show the dyndolod location like you did last night with the water.
